docs/configuration.asciidoc

```diff
@@ -750,6 +750,17 @@ To trace Django requests, the agent uses a middleware, `elasticapm.contrib.djang
 By default, this middleware is inserted automatically as the first item in `settings.MIDDLEWARES`.
 To disable the automatic insertion of the middleware, change this setting to `False`.
 
+[float]
+[[config-django-commands-exclude]]
+==== `django_commands_exclude`
+|============
+| Environment                            | Django                    | Default
+| `ELASTIC_APM_DJANGO_COMMANDS_EXCLUDE`  | `DJANGO_COMMANDS_EXCLUDE` | `runserver*,migrate,createsuperuser,\*shell*,testserver`
+|============
+
+By default, Elastic APM instruments Django management commands.
+You can supply a list of commands that should not be instrumented.
+To disable instrumenting of management commands, set it to `*`.
```

Comment:
What would you think about making this a whitelist (with globbing included)? That would also make this much safer. (More thoughts in the review body)

Reply:
A whitelist would definitely be the safest method, but it creates a lot of work for the user. Large Django projects can easily have dozens of management commands, and gain new ones if new 3rd party Django apps are added to the project. Keeping the white list up-to-date could be tedious work.

Another draw-back of a whitelist is that we can't provide a blacklist of commands that we should absolutely not instrument in code.

How about adding a second option, which is a flag, and disable it by default? Add some specific documentation in the Django docs, and include the `NOTE` there. That way, people will see the note when they read up on how to enable management command instrumentation.

Reply:
Sounds reasonable to me. 👍